settings/src/test/java/org/odk/collect/settings/ODKAppSettingsImporterTest.kt

```diff
@@ -0,0 +1,44 @@
+package org.odk.collect.settings
+
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Test
+import org.odk.collect.projects.InMemProjectsRepository
+import org.odk.collect.projects.Project
+import org.odk.collect.settings.support.SettingsUtils.assertSettingsEmpty
+
+class ODKAppSettingsImporterTest {
```

Comment:
Testing at this level is nice for checking specifics of our schema etc while also exercising the whole import "framework". I looked at adding tests for the enums like @grzesiek2010 suggested, but I backed out as it was going to be a lot of busy work. We don't currently have coverage for those, so I don't think it makes sense to spend any longer here adding that in.